partner_multi_relation/models/res_partner_relation_type.py

```diff
@@ -168,8 +158,41 @@ def get_category_condition(vals, side):
                                 relation.date_end > cutoff_date):
                             relation.write({'date_end': cutoff_date})
 
+    def update_right_vals(self, vals):
+        """
+        When you update the left values of a symmetric relationship,
+        make sure that the right ones are updated as well.
+        @attention: All fields ending in `_right` will have their values
+                    replaced by the values of the fields whose names end
+                    in `_left`.
+        """
+        right_vals = {}
+        name = vals.get('name')
+        if name:
+            right_vals['name_inverse'] = name
+        left_keys = filter(lambda key: key.endswith('_left'), vals.keys())
```

Comment:
Obsessive check , but possible
will cause problems if somebody has  a field that ends with _left  in vals other than the ones we want.  filtering explicitly contact_type_left and  partner_category_left   looks safer.

Reply:
@gfcapalbo Doing this in a generic way is needed to make sure that when extention modules add extra _left and _right fields, thses will be covered automatically. So this is quit intentional.

Reply:
ok. i had forwarned it was obsessive,
Second proposal. modularize this in a function "_get_left_keys" where the extending modules can add their left keys and protect against my . albeit , rare scenario.